backend/tournesol/admin.py

```diff
@@ -104,7 +104,17 @@ def get_language(obj):
 
 @admin.register(EntityCriteriaScore)
 class EntityCriteriaScoreAdmin(admin.ModelAdmin):
-    pass
+    list_display = (
+        'entity',
+        'criteria',
+        'score'
+    )
+    list_editable = (
+        'score',
+    )
```

Comment:
I am not convinced that make this `score` editable is useful, as they well be overridden on the next ml run.